### PR TITLE
Add link to Thompson paper and clarifications to HMI map example

### PR DESCRIPTION
**Pull request by dstansby**
- Add link to Thompson paper
- Actually check that CEA is the right projection by printing the keywords
- Clarify the CUNIT2 multiplication

### CODE REVIEW

**Comment by nabobalis** [comment]
This example in 1.0?

**Comment by nabobalis** [comment]
Merge it once the "bot" come back.

File: examples/plotting/hmi_synoptic_maps.py [hunk 1]
```diff
@@ -28,11 +28,21 @@
...
-# CUNIT2 should be in degrees and CDELT2 should be multiplied by 180/pi. Also
-# the value of CDELT1 has the wrong sign so let's fix this.
+# is actually the sine of latitude. This is a cyclindrical equal area (CEA)
+# projection, which can be checed by looking at the CTYPE keywords.
+print(syn_map.meta['CTYPE1'])
+print(syn_map.meta['CTYPE2'])
+
+###############################################################################
+# With reference to the Thompson (2006) paper
+# section 5.5 (https://doi.org/10.1051/0004-6361:20054262),
```
Comment (nabobalis) [inline examples/plotting/hmi_synoptic_maps.py]: Make this a RST link?

**Comment by dstansby** [comment]
Thanks for the changelog!

**Pull request merged**